mods/ra/audio/notifications.yaml

```diff
@@ -113,7 +113,9 @@ Speech:
 		WarningFourMinutesRemaining: 4minr
 		WarningFiveMinutesRemaining: 5minr
 		Win: misnwon1
-
+	AltTexts:
```

Comment:
I think it would make more sense to assign these to the individual notifications.  This then extends naturally in the future to our long-planned "sound sequences" that allow additional properties to be configured per notification. 

E.g.:
```yaml
 Speech:
	Notifications:
		BaseAttack: baseatk1
			TextNotifiation: Base under attack
```